Plugin/Catalog/Model/Sortby.php

```diff
@@ -102,7 +102,7 @@ public function afterGetAllOptions( // phpcs:ignore Generic.CodeAnalysis.UnusedF
         ) {
             // new option
             $customOption = [
-              ['label' => __(['Relevance']), 'value' => NostoHelperSorting::NOSTO_PERSONALIZED_KEY]
```

Comment:
Why is this not caught by Phan?

Reply:
Not sure why, need to look further into that